### ISSUE TITLE
Locator unusable

### ISSUE THREAD

**Issue by lehmannro** [OpenTechSchool/www.opentechschool.org#197]
While I admire the work that went into the new and shiny locator, there are several deficiencies with it currently:
- Scrolling down is hard.  On my screen size (standard 14") my mouse leaps into the map at some point, which is when the downward scroll is _caught_ and instead zooms the map.
- Names are confusing.  I'm not sure if visitors are caught up by the "CodeHub" prefix for Bristol, but "{Open}TechSchool" for Vienna is definitely confusing.
- Search is awkward.  Actually searching for a location is hard, because we always auto-detect the location.
- Selection is broken.  When I click on _Brussels,_ it shows Brussels _and_ Dortmund.  I presume that's because it's _close_ but, uh, I'm not sure commuting 3 hours for an OTS event is too commonplace.  This wouldn't be too bad if it weren't about the event list, which then gets conflated with events from a location I didn't even select.
- Bonus: Activity is hard to gauge.  Not sure if we want to see this directly, but it's not entirely obvious from the list that Stockholm and Melbourne are actually defunct right now.

**Comment by lehmannro** [comment]
Also:
- Links go to Meetup instead of our Web site.

**Comment by xMartin** [comment]
Yes :) I also don't see too much value in it. a simple map with markers would be sufficient maybe. I don't expect users to want a tool to find their closest events currently.

I'd opt for removing it.

**Comment by Ivoz** [comment]
I'm also confused by the listing of people in the 'Teams' section just underneath.
